### PR TITLE
Subscription status integration

### PR DESCRIPTION
**Pull request by alex-deaconu**
Added missing buttons

[stage-3]

### CODE REVIEW

File: test/e2e/editor/cases/presentation-add.js [hunk 1]
```diff
@@ -66,8 +66,10 @@ var PresentationAddScenarios = function() {
       presentationPropertiesModalPage.getNameInput().clear();
       presentationPropertiesModalPage.getNameInput().sendKeys(presentationName);
       helper.clickWhenClickable(presentationPropertiesModalPage.getApplyButton(), 'Apply Button');
+    });
 
-      workspacePage.getSaveButton().click();
+    it('should save presentation', function() {  
+      helper.clickWhenClickable(workspacePage.getSaveButton(), 'Save Button');
```
Comment (alex-deaconu) [inline test/e2e/editor/cases/presentation-add.js]: Trying to fix a test that was failing a couple of times.

File: web/partials/storage/storage-modal.html [hunk 2]
```diff
@@ -4,12 +4,20 @@
...
+  </div>
+
 	<div class="padding-large-horizontal padding-large-bottom">
 	<div class="app-header half-top">
 	  <div class="title-block">
 	    <!-- App Title -->
-	    <h1 id="title">Select a File</h1>
-	  </div>
+	    <h1 id="title" translate="storage-client.select-{{filesFactory.type}}"></h1>
+    </div>
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: Using i18n string, as per suggestion from the previous PR.

File: web/partials/storage/storage-modal.html [hunk 3]
```diff
@@ -80,5 +94,18 @@ <h1 id="title">Select a File</h1>
         </tbody>
       </table>
 		</div>	
+    <div ng-show="trialAvailable">
+      <ng-include src="'partials/storage/start-trial.html'" ng-show="trialAvailable"></ng-include>
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: Trial overlay; shows when trial is available.

File: web/partials/storage/storage-modal.html [hunk 4]
```diff
@@ -80,5 +94,18 @@ <h1 id="title">Select a File</h1>
...
+    </div>
+
+    <div ng-show="multipleFileSelector && !trialAvailable">
```
Comment (alex-deaconu) [inline web/partials/storage/storage-modal.html]: Multiple selector buttons. They have their own function (`fileSelectorFactory.sendFiles()`) and logic. Will try to refactor this when updating unit tests (next task).

File: web/scripts/storage/controllers/ctr-storage-selector-modal.js [hunk 5]
```diff
@@ -27,6 +28,13 @@ angular.module('risevision.storage.controllers')
         }
       });
 
+      $rootScope.$on('subscription-status:changed',
+        function (e, subscriptionStatus) {
```
Comment (alex-deaconu) [inline web/scripts/storage/controllers/ctr-storage-selector-modal.js]: Component emits this event.

File: web/scripts/storage/services/svc-storage.js [hunk 6]
```diff
@@ -49,6 +50,9 @@ angular.module('risevision.storage.services')
               return storageApi.startTrial(obj);
             })
             .then(function (resp) {
+              $rootScope.$emit('refreshSubscriptionStatus',
+                'trial-available');
```
Comment (alex-deaconu) [inline web/scripts/storage/services/svc-storage.js]: Event goes to `subscription-status-component`, which in turn updates UX to hide Trial overlay.

**Comment by alex-deaconu** [comment]
Please note this PR is to our existing `feature/storage-selector` branch.

@ezequielc please review. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good. :+1:  Thanks!

**Pull request merged**